src/api-models/fba-inventory-api-model/api.ts

```diff
@@ -76,23 +76,14 @@ export interface Granularity {
      * @type {string}
      * @memberof Granularity
      */
-    granularityType?: GranularityGranularityTypeEnum | 'Marketplace';
+    granularityType?: string;
```

Comment:
@nguyentoanit is this change also in the models or is it our script broke?

Reply:
@moltar: From my view, this change is also in the models. Because `enum` field was removed in `Granularity` object
- Before: https://github.com/amzn/selling-partner-api-models/commit/e3955b7f024b3703647288d8199ecacc0d4146e5#diff-be8161cc5a16190605a5b4e21ea1a060706650278cbfd47a8813173868e23726L34
- After: https://github.com/amzn/selling-partner-api-models/commit/e3955b7f024b3703647288d8199ecacc0d4146e5#diff-be8161cc5a16190605a5b4e21ea1a060706650278cbfd47a8813173868e23726R395

They have updated the docs too: https://github.com/amzn/selling-partner-api-docs/blob/main/references/fba-inventory-api/fbaInventory.md#granularity

Reply:
@moltar: From my view, this change is also in the models. Because `enum` field was removed in `Granularity` object
- Before: https://github.com/amzn/selling-partner-api-models/commit/e3955b7f024b3703647288d8199ecacc0d4146e5#diff-be8161cc5a16190605a5b4e21ea1a060706650278cbfd47a8813173868e23726L34
- After: https://github.com/amzn/selling-partner-api-models/commit/e3955b7f024b3703647288d8199ecacc0d4146e5#diff-be8161cc5a16190605a5b4e21ea1a060706650278cbfd47a8813173868e23726R395

They have updated the docs too: https://github.com/amzn/selling-partner-api-docs/blob/main/references/fba-inventory-api/fbaInventory.md#granularity

Reply:
How about this? Do you have any concern else?

Reply:
LGTM